mojap_metadata/converters/glue_converter/__init__.py

```diff
@@ -491,7 +538,34 @@ def generate_from_meta(
                 database_name, f"msck repair table {database_name}.{metadata.name}"
             )
 
-    def generate_to_meta(self, database: str, table: str) -> Metadata:
+    def generate_to_meta(
+        self,
+        database: str,
```

Comment:
again not actually linked to this change but there is an inconsistency in naming between `generate_to_meta` and `generate_from_meta` (`database` vs `database_name`). Would be nice to align but i guess that might be a breaking change for existing users so may be we just leave as is?

Reply:
I'm leaning towards leaving as it is since it would be a breaking change.